api/openshift.deploy.yml

```diff
@@ -210,6 +210,16 @@ objects:
                     secretKeyRef:
                       key: data-encryption-key
                       name: ${NAME}-${ZONE}-${COMPONENT}
+                - name: CLIENT_API_BASE_URL
```

Comment:
I'm not sure this needed to be a secret - the URL isn't a secret, but it does need to be configured for each environment. Given that for the object storage we stored everything as secrets, this is being consistent so should be fine.

Reply:
Basil, your sense is very sharp. Yes, initially I was troubled with it and was thinking I might need to make this one not a secret, but after looking into other existing secret, I glued them together for consistency. Hope it is okay, we can change it if you change mind.